ansible/bvt-slave.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Setup slave node for Build Verification Tests
```

Comment:
Let's remove all mentions to 'slave', this is a Jenkins-related concept. If I understand correctly, we will have to run both 'jenkins-slave' and 'bvt-slave' playbooks on a host to be able to use it in the pipeline, so I'd leave the "slave" setup to 'jenkins-slave' and use 'bvt-host' for this one.